pkg/storegateway/storegatewaypb/rpc.proto

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+syntax = "proto3";
+package gatewaypb;
+
+import "github.com/thanos-io/thanos/pkg/store/storepb/types.proto";
+import "store/labelpb/types.proto";
```

Comment:
This should be `"github.com/thanos-io/thanos/pkg/store/labelpb/types.proto" `but when added it causes a circular dependency inside of the thanos storepb/types.proto files. Any ideas on how to resolve this?

Reply:
I think we should just copy `github.com/thanos-io/thanos/pkg/store/labelpb/types.proto` too. Is it feasible?